src/components/CounterWithPrevious/CounterWithPrevious.tsx

```diff
@@ -1,28 +1,19 @@
-import React, { useState } from 'react';
 import { Button } from '../Button';
 import styles from './CounterWithPrevious.module.scss';
 import { usePrevious } from '@/hooks/usePrevious';
 
 export const CounterWithPrevious: React.FC = () => {
```

Comment:
```suggestion
export const CounterWithPrevious: React.FC = () => {
```
saya mendapatkan error `'React' is not defined`

Reply:
Sudah ditambahkan